src/index.js

```diff
@@ -1,18 +1,30 @@
-var Firebase = require('firebase');
+var firebase = require('firebase');
 
 /**
  * The Botkit firebase driver
  *
- * @param {Object} config This must contain a `firebase_uri` property
+ * @param {Object} config This must contain either a `firebase_uri` property (deprecated) or a `databaseURL` property
  * @returns {{teams: {get, save, all}, channels: {get, save, all}, users: {get, save, all}}}
  */
 module.exports = function(config) {
 
-    if (!config || !config.firebase_uri) {
-        throw new Error('firebase_uri is required.');
+    if (!config) {
+        throw new Error('configuration is required.');
     }
 
-    var rootRef = new Firebase(config.firebase_uri),
+    // Backwards compatibility shim
+    var configuration;
+    if (config.firebase_uri) {
+        configuration.databaseURL = config.firebase_uri;
+    } else if (!config.databaseURL) {
+        throw new Error('databaseURL is required.');
+    }   else {
+        configuration = config;
+    }
+
+    var app = firebase.initializeApp(config),
```

Comment:
I believe that you meant to call `initializeApp` with `configuration` not `config`